libs/core/src/components/sage-textarea/sage-textarea.scss

```diff
@@ -0,0 +1,128 @@
+    *
+    * @prop --border-color-default: Border color of the textarea text
+    * @prop --color-hover: Border color of the textarea text when hovered
+    *
+    * @prop --label-font-size: Font size of the label text
+    * @prop --label-font-weight: Font weight of the label text
+    * @prop --label-line-height: Line height of the label text
+    * @prop --label-margin-bottom: Bottom margin of the label
+    *
+    * @prop --field-font-size: Font size of the textarea text
+    * @prop --field-font-weight: Font weight of the textarea text
+    * @prop --field-line-height: Line height of the textarea text
+    * @prop --field-padding-block: Block padding for the textarea text
+    * @prop --field-padding-inline: Inline padding for the textarea text
+    *
+    * @prop --hint-font-size: Font size of the hint text
+    * @prop --hint-font-weight: Font weight of the hint text
+    * @prop --hint-line-height: Line height of the hint text
+    * @prop --hint-margin-top: Top margin of the hint
+    */
+
+  --error-color: #cf3c32;
+
+  --background: #fff;
+  --background-disabled: #f6f8f8;
+
+  --color: #202327;
+  --color-error: var(--error-color);
+  --color-placeholder: #60666c;
+  --color-disabled: #60666c;
+
+  --border-color-default: #d3d5d9;
+  --border-color-hover: #b5bac0;
+  --border-color-focus-visible: #eceeef;
+  --border-color-error: var(--error-color);
+
+  --focus-visible-outline-color: #8ecafb;
+  --focus-visible-outline-color-error: #fdb0aa;
+
+  --label-font-size: 16px;
+  --label-font-weight: 500;
+  --label-line-height: 24px;
+  --label-margin-bottom: 8px;
+
+  --field-font-size: 16px;
+  --field-font-weight: 400;
+  --field-line-height: 24px;
+  --field-padding-block: 8px;
+  --field-padding-inline: 16px;
+
+  --hint-font-size: 12px;
+  --hint-font-weight: 400;
+  --hint-line-height: 20px;
+  --hint-margin-top: 8px;
+
+  display: inline;
+}
+
+.sage-textarea {
+  display: flex;
+  flex-direction: column;
+}
+
+label {
+  display: block;
+  font-size: var(--label-font-size);
+  font-weight: var(--label-font-weight);
+  line-height: var(--label-line-height);
+  margin-bottom: var(--label-margin-bottom);
+}
+
+textarea {
+  border: 1px solid var(--border-color-default);
+  border-radius: 10px;
+  font-family: inherit;
+  font-size: var(--field-font-size);
+  font-weight: var(--field-font-weight);
+  line-height: var(--field-line-height);
+  padding: var(--field-padding-block) var(--field-padding-inline);
+
+  &:hover:not(:disabled) {
+    border-color: var(--border-color-hover);
+  }
+
+  &:disabled {
+    background-color: var(--background-disabled);
+    color: var(--color-disabled);
+    cursor: not-allowed;
+  }
+
+  &:focus-visible {
+    border-color: var(--border-color-focus-visible);
+    outline: 4px solid var(--focus-visible-outline-color);
+  }
+
+  &::placeholder {
+    color: var(--color-placeholder);
+    font-style: italic;
+  }
+
+  &:has(~ .sage-textarea__error-text) {
```

Comment:
Same note as above: This css doesn't work in FF yet but it is coming. It works in Firefox with flags enabled. Just something to keep an eye on for now I think.